### ISSUE TITLE
PlatformAnalysis cannot do single simulation Analysis

### ISSUE THREAD

**Issue by devclinton** [InstituteforDiseaseModeling/idmtools#1925]

**Comment by devclinton** [comment]
@shchen-idmod - Could you add some details here to describe the exact issue?

**Comment by shchen-idmod** [comment]
> @shchen-idmod - Could you add some details here to describe the exact issue?
Like if I want to test my analyzer with just one simulation, I can not do that:
```
    analysis = PlatformAnalysis(
        platform=platform, simulation_ids=["b816f387-cb04-eb11-a2c7-c4346bcb1553"],
        analyzers=[PopulationAnalyzer, AdultVectorsAnalyzer], 
        analysis_name="SSMT Analysis Simple 1")
    )
```
In SSMTWorkItem, we can pass related_simulations though

**Comment by devclinton** [comment]
@shchen-idmod @ZDu-IDM how often do users run single simulation experiments/analysis? 

I think @ZDu-IDM you mention it is to be compatible with AnalyzeManager. If so, and work is really small effort, maybe we can do as a low priority item. If users are not often using though, we could push back

**Comment by shchen-idmod** [comment]
The main purpose this to use single sim to test logic of analyzer. so good to have for testing especially since we have this ssmtworkitem already. But low pri definitely.

**Comment by shchen-idmod** [comment]
Fixed in #2149
